### PR TITLE
Les86 swagger email

### PR DESCRIPTION
**Pull request by ritamma**

### CODE REVIEW

File: app/views/users/check_email.jbuilder [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # frozen_string_literal: true
 
-json.email_exists @user.present?
+json.emailExists @user.present?
```
Comment (fooooxmr) [inline app/views/users/check_email.jbuilder:3]: Why did you change it? It's not a ruby way :) Use snake case here, please :)

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -25,6 +25,22 @@
       end
     end
   end
+  path '/{locale}/check_email?email={email}' do
+    parameter name: 'locale', in: :path, type: :string, description: 'locale'
+    parameter name: 'email', in: :path, type: :string, description: 'email'
+
+    get('check email') do
+      produces 'application/json'
+      response(200, 'successful') do
```
Comment (fooooxmr) [inline spec/requests/users_spec.rb:34]: Add example, please.

**Review by fooooxmr — CHANGES_REQUESTED** [review]

File: app/views/users/check_email.jbuilder (hunk 1 above)
Comment (ritamma) [reply app/views/users/check_email.jbuilder:3]: Sergey said that it's better for front-end team.

**Review by fooooxmr — APPROVED** [review]

**Review by evrusha — APPROVED** [review]

**Review by fooooxmr — APPROVED** [review]
